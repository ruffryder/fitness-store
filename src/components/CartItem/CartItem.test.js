import { shallow } from "enzyme";
import React from "react";
import CartItem from "./CartItem";

it("renders CartItem component", () => {
  const item = { imageUrl: "url", name: "sampleName", price: 40, quantity: 1 };
  expect(shallow(<CartItem item={item} />)).toMatchSnapshot();
});
