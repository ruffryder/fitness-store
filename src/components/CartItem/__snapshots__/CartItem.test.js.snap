// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CartItem component 1`] = `ShallowWrapper {}`;
